tox.ini

```
@@ -14,7 +14,8 @@ deps =
     dev: numpy>=0.0.dev0
     dev: scipy>=0.0.dev0
     dev: astropy>=0.0.dev0
-    dev: git+https://github.com/astropy/specutils.git
+    dev: specutils>=0.0dev0
```

```suggestion
    dev: git+https://github.com/astropy/regions.git
    dev: git+https://github.com/astropy/specutils.git
```
Revert to see if it works against the 2.0 branch now, and give regions dev a try for the macOS build.